src/dotnet/commands/dotnet-install/dotnet-install-tool/ProjectRestorer.cs

```diff
@@ -47,6 +47,7 @@ public ProjectRestorer(IReporter reporter = null)
             {
                 "--runtime",
                 GetRuntimeIdentifierWithMacOsHighSierraFallback(),
+                "--ignore-failed-sources", //or it will throw when a feed cannot be reached
```

Comment:
It would be helpful to say *why* it would throw in this case: It's because we're getting wildcard version as @livarcocc just confirmed. Not having used wildcards much, I had never heard of --ignore-failed-sources.

Reply:
I will add more to the comment. It is simply the default behavior. And `--ignore-failed-sources` is a flag CLI added to restore. And it ends up passing to nuget as `/p:RestoreIgnoreFailedSource=true`